doc/classes/AudioStreamPlayer.xml

```diff
@@ -44,6 +44,17 @@
 				Plays a sound from the beginning, or the given [param from_position] in seconds.
 			</description>
 		</method>
+		<method name="play_scheduled" experimental="">
+			<return type="void" />
+			<param index="0" name="absolute_time" type="float" />
+			<param index="1" name="from_position" type="float" default="0.0" />
+			<description>
+				Schedules a sound to be played on the [AudioServer]'s timeline at [param absolute_time] in seconds (see [method AudioServer.get_absolute_time]). If [param absolute_time] is in the past, the sound will be played immediately. The sound starts from the given [param from_position] in seconds.
+				Use this method for high precision playbacks, such as a metronome or other rhythm-based sounds.
+				[b]Note:[/b] By default, calling this method will cut off the currently playing sound on this player. Consider changing [member max_polyphony] to prevent this.
```

Comment:
This is true, but in theory this can happen always, not just when `max_polyphony` is `1`.
```suggestion
				[b]Note:[/b] Calling this method will cut off the oldest sound playing on this node. Consider increasing [member max_polyphony] to prevent this.
```
or
```suggestion
				[b]Note:[/b] Calling this method after [member max_polyphony] is reached will cut off the oldest sound playing on this node.
```
or similar.